remoulade/api/main.py

```diff
@@ -76,6 +86,23 @@ def get_actors():
     return {"result": [actor.as_dict() for actor in get_broker().actors.values()]}
 
 
+@app.route("/groups")
+def get_groups():
+    args = PageSchema().load(request.args.to_dict())
+    backend = remoulade.get_broker().get_state_backend()
+    groups = defaultdict(list)
+    states = backend.get_states()
+    if args.get("search_value"):
+        keys = ["message_id", "name", "actor_name", "args", "kwargs"]
+        value = args["search_value"].lower()
+        states = [state for state in states if state.group_id and dict_has(state.as_dict(), keys, value)]
+    for state in states:
+        if state.group_id:
+            groups[state.group_id].append(state.as_dict())
+    groups = [{"group_id": group_id, "messages": messages} for group_id, messages in groups.items()]
```

Comment:
maybe find a way to not serialize args/kwargs in reponse as they are not shown in app.